charts/gardener-dashboard/values.yaml

```diff
@@ -32,38 +36,51 @@ resources:
   requests:
     cpu: 100m
     memory: 128Mi
-hosts:
-  - dashboard.ingress.example.org
-  - dashboard.example.org
+
 ingress:
   annotations:
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
     nginx.ingress.kubernetes.io/use-port-in-redirects: "true"
     kubernetes.io/ingress.class: nginx
-tls:
-  crt: |
-    -----BEGIN CERTIFICATE-----
-    Li4u
-    -----END CERTIFICATE-----
-    -----BEGIN CERTIFICATE-----
-    Li4u
-    -----END CERTIFICATE-----
-  key: |
-    -----BEGIN RSA PRIVATE KEY-----
-    Li4u
-    -----END RSA PRIVATE KEY-----
+  # # configuration of hosts used for rules and tls
+  # hosts:
+  # - dashboard.ingress.example.org
+  # - dashboard.example.org
+  tls:
+    secretName: gardener-dashboard-tls
```

Comment:
shouldn't this also be commented out by default?

Reply:
This is the default name for the secret. If I remove this default value the `.Values.ingress.tls` property will be falsy. It would need to look like this:
```
  tls: {}
    # secretName: gardener-dashboard-tls
```
I think it would be anyway better to specify all default values in `value.yaml`. What do you think.

Reply:
It is required now. If not changed the default value from `values.yaml` will be used.